tests/phpunit/tests/test-canonical.php

```diff
@@ -50,6 +50,16 @@ function( $taxonomy ) {
 				}
 			}
 		);
+		$test = register_post_type(
```

Comment:
```suggestion
		register_post_type(
```

Reply:
Forgotten after debugging and verify everything is ok with registering the post type 🙂

Done in 6b8770cdc399302902e14c7e24a74b29ffe243ac